packages/noya-state/src/reducers/layerReducer.ts

```diff
@@ -107,9 +172,18 @@ export function layerReducer(
         ids.includes(layer.do_objectID),
       );
 
-      // We delete in reverse so that the indexPaths remain accurate even
-      // after some layers are deleted.
+      const symbolsInstancesIndexPaths = getSelectedSymbols(
```

Comment:
This `deleteLayer` action looks very similar to `deleteSymbol`. Can we remove `deleteSymbol` entirely, and just dispatch this action instead?

Reply:
The main difference is that in the deleteSymbols goes trough every page and the deleteLayer only goes trough the main page. How should I go about changing this part? Do I make it general

Reply:
Ah right. I think it's OK to go through every page in `deleteLayer`. So we can move the `deleteSymbols` implementation into `deleteLayer`, and remove `deleteSymbols`.

I want to move away from relying on the current page in actions in general, so I like that approach better anyway. A lot of the actions that use the current page were some of the first ones I wrote, but it's less flexible than searching every page